lib/acme/client/util.rb

```diff
@@ -0,0 +1,59 @@
+module Acme::Client::Util
+  def urlsafe_base64(data)
+    Base64.urlsafe_encode64(data).sub(/[\s=]*\z/, '')
+  end
+
+  # Sets public key on CSR or cert.
```

Comment:
Are the methods below used outside of tests? If not I'd put them in a module in `spec/support`